docs/ProgrammingCryptol/highAssurance/HighAssurance.tex

```diff
@@ -174,6 +174,38 @@ \subsection{Capturing test vectors}
   sure the tweak to the above example (removal of dummy single formal
   parameter) works.}
 
+%~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+\subsection{Generating test vectors}
+\label{sec:gentestvec}
+
+Cryptol has the ability to generate test vectors for non-polymorphic functions (there is limited support for polymorphic types) using the \texttt{:dumptests} REPL command.
+When \texttt{:dumptests} is given a file to dump the output and a function in scope, it will randomly generate inputs for the function and run them for you.
+\begin{Verbatim}
+  :dumptests <results file> <function>
+  :dumptests results.txt myFunction
+\end{Verbatim}
+The results file will contain a tab delimited file in which the first column contains the output, and each column after contains the inputs to the function in order.
```

Comment:
suggested rephrase: "tab delimited file" -> "tab delimited table"? I don't love the phrasing of "the file contains a file" but I'm struggling to find a better word.